packages/client/src/object/convertWireToOsdkObjects.ts

```diff
@@ -112,6 +116,80 @@ export async function convertWireToOsdkObjects(
   return ret;
 }
 
+export async function convertWireToOsdkObjects2(
```

Comment:
Added a new method for now, once we remove the old api functionality, this can be cleaned up to just be this one method.

Reply:
The new APIs return all objects with their object prop names, but scoped down to the interface. We also get a map that tells us how to convert from the object props to their respective SPTs